CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 ### 2.0.2 (Next)
 
 * [#92](https://github.com/mongoid/mongoid-locker/pull/92): Remove TTL and update unique index definitions - [@cesarizu](https://github.com/cesarizu).
-* Your contribution here.
+* [#92](https://github.com/mongoid/mongoid-locker/pull/97): Add support for mongoid 8.0.x versions - [@randikabanura](https://github.com/randikabanura)
```

Comment:
Add a period in the end. 

I would just say "Added support for Mongoid 8" (we tend to use Added/Updated... should fix other places too to be consistent).

Reply:
Nice. I have updated the other locations that I could find with the inconsistency.